app/components/access_panels/location_item_component.rb

```diff
@@ -18,11 +18,15 @@ def availability_text
     end
 
     def current_location_text
-      return if item.effective_location&.details&.key?('availabilityClass') || item.effective_location&.details&.key?('searchworksTreatTemporaryLocationAsPermanentLocation')
+      return if has_avaliability_class? || item.effective_location&.details&.key?('searchworksTreatTemporaryLocationAsPermanentLocation')
 
       item.current_location.name
     end
 
+    def has_avaliability_class?
+      item.effective_location&.details&.key?('availabilityClass')
```

Comment:
Is this any availability class data, or should we limit it just to the in-process  value? I think we still want to do live lookups for e.g. stuff with "off-site", no?

Reply:
I was understandings that if it had an availability class we did not live lookup. @saseestone do you know?

Reply:
The availability class values are:

```
Available
Unavailable
In_process
Offsite
```
(From https://github.com/sul-dlss/folio-graphql/blob/501e6a891e8f6b586ab4bfc8ded6e1ca2f8148e6/src/schema.graphql#L4365-L4370)

Offsite is not like the others and requires special handling (because it could be paged to campus and be checked out/lost/etc)